src/containers/profile.js

```diff
@@ -5,44 +5,96 @@ import Splitads from "../components/profile/splitads";
 import ProfileHeader from "../components/profile/profileHeader";
 import Video from "../components/profile/video";
 import Singlerowads from "../components/profile/singlerowads";
+import helpers, { getcontestant, getStyle } from "../utils/helpers";
 
 export default class Profile extends Component {
-  render() {
-    const myStyle = {
-      backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url(
-        https://images.pexels.com/photos/167446/pexels-photo-167446.jpeg?cs=srgb&dl=adult-beard-blur-167446.jpg&fm=jpg
-      )`
-    };
-    const myStyleProfile = {
-      backgroundImage: `url(
-            https://images.pexels.com/photos/167446/pexels-photo-167446.jpeg?cs=srgb&dl=adult-beard-blur-167446.jpg&fm=jpg
-          )`
+  constructor(props) {
+    super(props);
+    this.state = {
+      fetching: true,
+      profilePicture: null,
+      country: null,
+      state: null,
+      firstName: null,
+      lastName: null,
+      profileUrl: null,
+      bio: null,
+      videoUrl: null,
+      uniqueCode: null,
+      show: false
     };
+    this.handleClick = this.handleClick.bind(this);
+  }
+  async componentDidMount() {
+    const {
+      match: { params }
+    } = this.props;
+
+    const response = await getcontestant(params.uniqueCode);
+
+    if (response) {
+      this.setState(prevState => ({
+        ...prevState,
+        ...response,
+        fetching: false
+      }));
+    }
+  }
+  handleClick() {
+    this.setState(prevState => ({ show: !prevState.show }));
+  }
+  render() {
+    const {
+      fetching,
+      profilePicture,
+      country,
+      state,
+      firstName,
+      lastName,
+      profileUrl,
+      bio,
+      videoUrl,
+      show
+    } = this.state;
+
+    const myStyle = getStyle("myStyle", fetching, profilePicture);
+    const myStyleProfile = getStyle("myStyleProfile", fetching, profilePicture);
+
     return (
       <div className="profile--container">
         <ProfileHeader
           myStyle={myStyle}
-          name={"Jessie Schwartz"}
-          state={"San Francisco CA"}
-          country={"United States"}
-          profilelink={"google.com"}
+          state={state || "San Francisco CA"}
+          country={country || "United States"}
+          profilelink={profileUrl || "google.com"}
+          firstName={firstName || "Jessie"}
+          lastName={lastName || "Schwartz"}
+          bio={bio || helpers.mybio}
         />
         <div className="profile--body">
           <div className="row">
             <div className="col-sm-1" />
             <div className="col-sm-3">
               <Bio
                 myStyle={myStyleProfile}
-                name={"Jessie Schwartz"}
-                state={"San Francisco CA"}
-                country={"United States"}
-                profilelink={"google.com"}
+                state={state || "San Francisco CA"}
```

Comment:
You can just add the defaults to the states in the constructor, instead of using null there, and specifying another value here. State will update